azuredevops/internal/service/core/data_team.go

```diff
@@ -51,6 +51,10 @@ func DataTeam() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"top": {
+				Type:         schema.TypeInt,
+				ValidateFunc: validation.IntAtLeast(1),
+			},
```

Comment:
Add a default  value for `top`  (`Default: 100,`) in case user not specified. Otherwise the the hard coded value `2147483647` this will cause performance issue.

```suggestion
			"top": {
				Type:         schema.TypeInt,
				Optional:     true,
				Default:      100,
				ValidateFunc: validation.IntAtLeast(1),
			},
```